### PR TITLE
feat(KSM-4523, api/api-server.js, package.json, yarn.lock):

### PR DESCRIPTION
**Pull request by Jmg047**
Two endpoints:  /get-user/:id and /set-user/:id have been finished. no other significant changes otherwise. To start both `server.js` and `api-server.js` use `yarn start`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@reduxjs/toolkit": "^1.9.3",
     "@types/react-dom": "^18.0.11",
     "babel-eslint": "^10.1.0",
+    "concurrently": "^8.0.1",
```
Comment (pamelagilmour) [inline package.json:11]: I like this concurrently package is cool!

**Review by pamelagilmour — APPROVED** [review]
I see your React app rendering. Great job!
After looking at your code in api/api-server.js, I just navigated to localhost:3001/get-user/1 and got a successful response. Great job again! 

In the future it might be helpful for users/reviewers to have testing criteria that has a little bit more detail in your PR, essentially a way for us to prove your code is working as expected. In this case I didn't know what the parameter `id` could be, but I just picked a random number - 1 and it works. LGTM!

**Review by Chocoloco123 — COMMENTED** [review]
Overall it looks pretty good, nice job! I also got a 200 response. I'd love to sync up and walk through the code to make sure we're all on the same page about the data output we're looking for.

**Comment by Jmg047** [comment]
To test, 

``/get-user``
head over to `http://localhost:3001/get-user/`in Chrome (or any browser)  and append [0-1] after `get-user/` to view the user array data. 

``/set-user``
in Postman paste `http://localhost:3001/set-user/0`in the URL field make sure the request is set to `POST`, from there when `Send` is pushed the user at index 0 should display.

To create a new user, pass In the keys `firstName` and `lastName` and set the value to any string, the data should display when appending any number greater than 2 (ex .../set-user/5)

**Review by pamelagilmour — APPROVED** [review]
Looking good, works as expected.

**Review by Chocoloco123 — APPROVED** [review]
Hey Jaime, nice job with this PR. It’s looking like it’s working out well! I see that you implemented an array to hold your sample data, that’s fantastic! As we’re considering which is the best data type to store information, we also want to consider the size of the data it’ll hold. For the purposes of this exercise I think it’s great that you have an array here. In small data sets such as this it works very quickly. Though I also want to point out that as the data becomes larger, the time complexity of manipulating data in an object could slow. That’s why for larger sets of data, using objects is a great way to store our data. Here’s an article that talks about the differences in time complexity for arrays and objects that would be a good read. It also has some good examples as well. https://soshace.com/comparing-data-structures-in-javascript-arrays-vs-objects/ Please feel free to reach out if you have any questions or want to bounce ideas, we’re here to work with you and to help you grow as an engineer. Heck I might even create a ticket to explore time and space complexity, or maybe we’ll have a talk about it. And if you’re already feeling comfortable with time and space complexity, then you’re well on your way to building the skills you’ll need as an engineer in the field. Nice work!

**Pull request merged**